examples/example3/example3_withmanufactured.ipynb

```diff
@@ -0,0 +1,679 @@
+   "source": [
+    "## Generate model\n",
+    "Next we generate the model, which consists of four containers - compartments, species, reactions, and parameters.\n",
+    "\n",
+    "### Compartments\n",
+    "As described above, the two compartments are the cytosol (\"Cyto\") and the plasma membrane (\"PM\"). These are initialized by calling:\n",
+    "```\n",
+    "compartment_var = Compartment(name, dimensionality, compartment_units, cell_marker)\n",
+    "```\n",
+    "where\n",
+    "- name: string naming the compartment\n",
+    "- dimensionality: topological dimensionality (i.e. 3 for Cyto, 2 for PM)\n",
+    "- compartment_units: length units for the compartment (um for both here)\n",
+    "- cell_marker: integer marker value identifying each compartment in the parent mesh"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8ec50252",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Cyto = Compartment(\"Cyto\", 3, um, 1)\n",
+    "PMleft = Compartment(\"PMleft\", 2, um, 10)\n",
+    "PMright = Compartment(\"PMright\", 2, um, 11)\n",
+    "PMfront = Compartment(\"PMfront\", 2, um, 12)\n",
+    "PMback = Compartment(\"PMback\", 2, um, 13)\n",
+    "PMbottom = Compartment(\"PMbottom\", 2, um, 14)\n",
+    "PMtop = Compartment(\"PMtop\", 2, um, 15)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "12b4d3d6",
+   "metadata": {},
+   "source": [
+    "Create a compartment container."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "44349455",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cc = CompartmentContainer()\n",
+    "cc.add([PMleft, PMright, PMfront, PMback, PMtop, PMbottom, Cyto])"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "105b9b7e",
+   "metadata": {},
+   "source": [
+    "### Species\n",
+    "In this case, we have a single species, \"A\", which exists in the cytosol. A single species is initialized by calling:\n",
+    "```\n",
+    "species_var = Species(\n",
+    "            name, initial_condition, concentration_units,\n",
+    "            D, diffusion_units, compartment_name, group (opt)\n",
+    "        )\n",
+    "```\n",
+    "where\n",
+    "- name: string naming the species\n",
+    "- initial_condition: initial concentration for this species (can be an expression given by a string to be parsed by sympy - the only unknowns in the expression should be x, y, and z)\n",
+    "- concentration_units: concentration units for this species (μM here)\n",
+    "- D: diffusion coefficient\n",
+    "- diffusion_units: units for diffusion coefficient (μm<sup>2</sup>/sec here)\n",
+    "- compartment_name: each species should be assigned to a single compartment (\"Cyto\", here)\n",
+    "- group (opt): for larger models, specifies a group of species this belongs to;\n",
+    "            for organizational purposes when there are multiple reaction modules"
```

Comment:
```suggestion
    "In this case, we have a single species, \"A\", which exists in the cytosol."
```
We already have this written down in other demos, so I don't see the need for it here.

Reply:
Forgot to remove and update text in this reworked example, apologies!